app/javascript/components/patient/assessment/actions/SymptomOnset.js

```diff
@@ -84,7 +84,7 @@ class SymptomOnset extends React.Component {
                   this.props.calculated_symptom_onset ? moment(this.props.calculated_symptom_onset).format('MM/DD/YYYY') : 'blank'
                 }.`}
           </p>
-          {symptom_onset_invalid && (
+          {this.props.patient.isolation && !this.props.patient.symptom_onset && !this.props.symptomatic_assessments_exist && this.props.num_pos_labs === 0 && (
```

Comment:
I'm seeing this warning be displayed when I am in isolation and have no positive labs or symptom onset date and am trying to then set a symptom onset date.  that doesn't seem quite right because I am resolving the problem by adding a symptom onset date?

Reply:
great catch: https://github.com/SaraAlert/SaraAlert/pull/950/files#diff-c70b6a432860cf1cff1aa3bc624a2b72da37531257c62855dba8ee265610f6e9R87